api/models/categories.model.js

```diff
@@ -0,0 +1,23 @@
+const db = require("../../db/connection");
+const fs = require('fs/promises');
+
+exports.selectCategories = () => {
+    
+  return db
+    .query(
+      `
+    SELECT * from categories
+    `
+    )
+    .then((response) => {
+        return response.rows
+    });
+}
+
+exports.selectApi = () => {
```

Comment:
we don't need a model for task 3.5 - we can pop this functionality in the controller :)